Sources/Swift/Protocol/SentryViewControllerBreadcrumbTracking.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+@objc
+public protocol SentryViewControllerBreadcrumbTracking: NSObjectProtocol {
+
+    var screenName: String { get }
+}
```

Comment:
Thanks @m1entus for this PR.

Since this is a new public protocol, im just missing a documentation in here.

```suggestion
///
/// Use this protocol to customize the name used in the automatic
/// UIViewController performance tracker.
///
@objc
public protocol SentryViewControllerBreadcrumbTracking: NSObjectProtocol {

    /// The custom name of the UIViewController
    /// that will be used for the transaction name.
    var screenName: String { get }
}
```

Something like this.